app/controllers/students/base_controller.rb

```diff
@@ -8,7 +8,7 @@ def must_be_student
   end
 
   def current_team
-    @current_team ||= current_user.teams.first
+    @current_team ||= current_user.teams.last
```

Comment:
This could also be problematic if you allow to create a team in the next season or so. Maybe there is a way to actually get the team from the current season? Would something like this `current_user.teams.where(season: Season.current).first` work?

Also it would be great if you could add a test for this.